### ISSUE TITLE
[trackeros] Test search in Trackeros => Found no results while trying to browse this tracker (Test)

### ISSUE THREAD

**Issue by Nonii07** [Jackett/Jackett#12807]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Starting Jackett v0.20.285
Environment version: 6.0.1 (C:\ProgramData\Jackett\)
OS version: Microsoft Windows NT 10.0.19042.0 (64bit OS)
Jackett variant: CoreWindows
Running in Docker: No
```

### Description

they are 4 days updating the tracker, today tracker was up, but since the update, don't find anithing

### Logged Error Messages

System.Exception: Test search in Trackeros => Found no results while trying to browse this tracker
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in D:\a\1\s\src\Jackett.Common\Services\IndexerManagerService.cs:line 309
   at Jackett.Server.Controllers.IndexerApiController.Test() in D:\a\1\s\src\Jackett.Server\Controllers\IndexerApiController.cs:line 132
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in D:\a\1\s\src\Jackett.Server\Middleware\CustomExceptionHandler.cs:line 61

**Comment by garfield69** [comment]
The site has probably upgraded their unit3d to a new version and the indexer needs to be refreshed.
Please turn on the Jackett **enhanced** logging and repeat the test and upload the log including the Debug and HTML and hopefully with that info we can determine what version of unit3d they are now on and what change we have to make to the indexer.
See https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket
OR
If you send an invite to garfieldsixtynine -at- gmail.com I will do the rest.
Thanks.

**Comment by garfield69** [comment]
From the login page it appears they have upgraded to unit3d 5.3.0
So if you cannot send an invite, then actually a dump of the torrent search page will be the better option, just need to see if the categories have changed.
See https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-html-source
Thanks.

**Comment by Nonii07** [comment]
remember_web_59ba36addc2b2f9401580f014c7f58ea4e30989d=eyJpdiI6InBIbzBqKzRFSEZwZ3IyNC96MmlPSVE9PSIsInZhbHVlIjoiUFFqRHprZmlQL0FCalhNLy9uOFViYys5dUo0aE0rY2NLTFBvNTB4UmN2QVFGRU5rSGtzN3dtKzlaRVFmL2FPOHVrRm92azBOMlFHVW1KZU1sV0dlR2pUVFVyVUhUK3A5RXA0Z3ZPenRYakoxUE5aeUhvVWd0cXJleGdhdUpkRWNVTGRPa2NFUERYR3pMYTFEcmREaUtTMGtEU21jRHNhYlZHQ1I1Rk1PUUtOYjRObXBMM3BwUlJnR0M1dERDQTZrOTA0bHN6d2xVQXp6SDJWbDd2VXdaeGhYUE93c1dqSWJWVG96OVo3eHdHVT0iLCJtYWMiOiI0N2JjNDM1OWRmZjY4ZjAxOTczZDliZjk4NGFkMDE3ZGRhNzY1YTUxMGE3MmNhMjk0MDhiMWJhMTE4ZjQ4N2E1In0=; laravel_cookie_consent=1; XSRF-TOKEN=eyJpdiI6IkJ4Um5qYkhQU3hqTG9lYnFrd1g1Unc9PSIsInZhbHVlIjoidzVabndaZndqTzM5OEZ1bDRTWVNxN0kreXdzV1dXVE42N3dZenZJejM5OW1hU1dLWGxsWFpDNERqREJFVnByek5UeGRwQTN1RzJWcEpnRFRKbC9aSUFjeFE4ZnpWT29yeEpLRldCL3hCZmRiQWRkVCtKcHVCdHBWYnFCa3p4dVEiLCJtYWMiOiIzMzhlOTUzY2MxNGQyZTlhM2UwZDllMWI2NjBiMmUyZmIxNDM5MWIzMmRmMDNmZGIwMzYzNGRlYWRlM2RiZGIxIiwidGFnIjoiIn0=; laravel_session=eyJpdiI6IlZyZzlaQVFrVFN3eVBOcVV2cUZWa1E9PSIsInZhbHVlIjoiUS9DZ2Z4SWVqYjFuRGMwdmtpbGN2ZGUyMWxGNFdyc0VEU0ZIdWdlOG5rQmk2dUVOMUh2Y3F4M2FzcW9uVE9xMjlUSW1ISEtqV0dCMVUxZEIreXptMjJUMmdHSW04UXZtR1VIY3JqdkhMblhKN3Jpc2RRcUh0WW5oeUhuN0VoM1UiLCJtYWMiOiIyNjg2MDlhZTQ5M2M1MDdhMDhhMDZhMDQzZjg0MzM0YTEwMDhlNzA4ZTNlMjYzZTFkYjgwMDM1Y2MwNjgyOGI3IiwidGFnIjoiIn0=; io=toD_RAvNXDDu8-UlAAM-

**Comment by garfield69** [comment]
huh?

**Comment by Nonii07** [comment]
if you need something more, feel free to ask me

thanks for fast responce

**Comment by garfield69** [comment]
I'm sorry, I don't understand what I'm supposed to do with that cookie :confused:

I simply need to to login to trackeros, navigate to the torrents search page, and dump the html using the method at https://github.com/Jackett/Jackett/issues/12807#issuecomment-1010596664

**Comment by Nonii07** [comment]
is what im doing, and the tracker drop me that info:

laravel_cookie_consent=1; io=tnmaAWVAHtGdowLzAAN1; XSRF-TOKEN=eyJpdiI6IjJNMUhMZURnUzVtRHRFUjMxZFVUaVE9PSIsInZhbHVlIjoidDd1Z2NKY1JiaFhMeDJpQk5YamNoQ2h4OUxNa0JZRFFqQ0pIV24yc3hZeW1SdEd3Nk1Md2lqRnFMcHRsNFYvdlh6VjdXQ0xUcjIwYm5CWWV0R3lYY043bmZVREMvL2hFUG0vaGxhNitOUWNvTFBnZTBFbEliVWQzZ2xzVTQ5YXEiLCJtYWMiOiI0Yjc3MTQ2MjFlNTdkOWQyMTVlZTYyNDVkMjg2Y2RjNWIzYmU1M2Q2ZTNjN2Y5MjBiOGU0MmRkYjMyN2M2ZjAzIiwidGFnIjoiIn0=; laravel_session=eyJpdiI6IktaTDh6S3RxaWVZWTVodENWeGlKOVE9PSIsInZhbHVlIjoia2x1WndwQ00xQ1BYT3hHM3c3VENRa2tSWkRxMG44SWdWd1ZXTStKT3NMbHoxWHQza0k3UkdQQWNXRzRkaVZhVGlDZlphOHcvalhUckpuNm5LY1pIR3l1aERhMHFJZi9MeTgza0hHTHluT0hvZG04ekIxMHJyR1NhVmhWekJIZFoiLCJtYWMiOiI0ZTBkYjM5NDYyZGQ4ZjUwODZmOWZiMWQ3NDc5OTEyZjkwZmMwMThjZWNhMmExNDQzYTlmYzE4N2ZkNTliZTYyIiwidGFnIjoiIn0=

**Comment by garfield69** [comment]
weird.
So using your browser, at the trackeros torrent search page, can you right click and select the `view page source`, select all and copy paste that instead?

**Comment by garfield69** [comment]
closed as duplicate of #12808
